service/src/main/java/bio/terra/workspace/service/workspace/CloudSyncRoleMapping.java

```diff
@@ -71,6 +71,7 @@ public class CloudSyncRoleMapping {
               "artifactregistry.tags.update",
               "cloudbuild.builds.create",
               "cloudbuild.builds.update",
+              "compute.instances.getGuestAttributes",
               "iam.serviceAccounts.get",
```

Comment:
Fair, let me make sure you can directly attach a compute permission to a notebook instance, since thats what it looks like the code is doing

Reply:
Tested out adding this directly to the AI_* roles, and it doesn't seem to work unfortunately. I've been Googling around, but I can't quite find a guide on how to determine if a permission will work if attached directly to a resource. In most cases its obvious, IE a storage* permission will work on a GCS bucket but not on a bigquery table. This is a case where its not so obvious, but I'm guessing it's not working because we're attaching a compute* permission to a notebook resource. I'll ask around internally to see if anyone else has any info here.

Reply:
It looks to me like GCP has copied over compute.instances* permissions to notebook.instances* permissions, but hasn't done all of them. I imagine if there was a notebooks.instances.getGuestAttributes this would work, but the API call goes directly to the compute API, not the notebooks API.

Reply:
thanks for the research.

Reply:
Thanks Yu - so this PR has a failure that I assume isn't related, what do y'all do here?

Reply:
i'm not admin of this repo. you can try rerun it and cross your finger or ask @jmczerk or @gjuggler to overrule the branch protection